Source/Charts/Utils/Platform+Accessibility.swift

```diff
@@ -58,13 +58,13 @@ open class NSUIAccessibilityElement: UIAccessibilityElement
     {
         get
         {
-            return super.accessibilityFrame
+            let frame = super.accessibilityFrame
+            return containerView?.convert(frame, to: UIScreen.main.coordinateSpace) ?? frame
```

Comment:
Maybe you can consider using our 'NSUIView' in platform.swift or similar what we did in platform+Color.swift for macOS and iOS, for `containerView?.convert()` to work on both iOS and macOS?

Reply:
The macOS part is using different calculations of the `accessibilityFrame` as it's already addressing this problem which I am trying to fix inside the iOS part. I believe it's because the bug on the macOS was more visible (as it occurred right after moving/resizing the application window). 

Here is what's done for macOS: 

```
    open var accessibilityFrame: NSRect
    {
        get
        {
            return accessibilityFrame()
        }

        set
        {
            guard let containerView = containerView else { return }

            let bounds = NSAccessibility.screenRect(fromView: containerView, rect: newValue)

            // This works, but won't auto update if the window is resized or moved.
            // setAccessibilityFrame(bounds)

            // using FrameInParentSpace allows for automatic updating of frame when windows are moved and resized.
            // However, there seems to be a bug right now where using it causes an offset in the frame.
            // This is a slightly hacky workaround that calculates the offset and removes it from frame calculation.
            setAccessibilityFrameInParentSpace(bounds)
            let axFrame = accessibilityFrame()
            let widthOffset = abs(axFrame.origin.x - bounds.origin.x)
            let heightOffset = abs(axFrame.origin.y - bounds.origin.y)
            let rect = NSRect(x: bounds.origin.x - widthOffset,
                              y: bounds.origin.y - heightOffset,
                              width: bounds.width,
                              height: bounds.height)
            setAccessibilityFrameInParentSpace(rect)
        }
    }
```

Because of this, I am thinking that with this PR both - macOS and iOS will work just fine as macOS was already fixed and this PR will fix iOS.

Thank you and I am sorry for the late answer.

Reply:
@liuxuan30 would you mind reviewing this, please?